modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -115,13 +115,73 @@ authService.login().then(() => {
 IAuthService authService = MobileCore.Instance.GetService<IAuthService>();
 
 var authOptions = DependencyService.Get<IAuthenticateOptionsProvider>().GetOptions();
-authService.Authenticate(authOptions).ContinueWith(user =>
-{
-    // Login success
+var user = await authService.Authenticate(authOptions);
+----
+****
+
+== Refreshing the authentication token
+
+Every time a user is authenticated, both an access and a refresh token get generated. They both have a lifespan that can be configured on the authentication server.
```

Comment:
When a user is authenticated, an access token and a refresh generated by the authentication server. See <link> for configuring the lifespan of these tokens.

Do we have have a link we can share with users?

Reply:
is this a good link?  https://www.keycloak.org/docs/3.2/server_admin/topics/sessions/timeouts.html